README.md

```diff
@@ -7,7 +7,7 @@ host:
 sudo apt-get install docker
```

Comment:
Our primary usage instruction should be:

```
docker pull kartoza/geogig
sudo docker run --name="geogig" -p 38080:8182 -d kartoza/geogig
```

and the rest shoul be for the hard core.